rosco-web/config/packer/gce.json

```diff
@@ -4,6 +4,7 @@
     "gce_account_file": "",
     "gce_zone": null,
     "gce_network": null,
+    "gce_subnetwork": "",
```

Comment:
Why is subnetwork initialized with `""` rather than `null` like the other `String` parameters above?